src/AppBundle/Controller/EditCounterController.php

```diff
@@ -293,4 +279,114 @@ public function latestglobalAction(Request $request, $project, $username)
             'ec' => $this->editCounter,
         ]);
     }
+
+
+    /**
+     * Below are internal API endpoints for the Edit Counter.
+     * All only respond with JSON and only to requests from the localhost
+     * (see access_control in security.yml).
```

Comment:
`see access_control in security.yml` -- this isn't actually implemented yet. I want to test this feature in a prod environment first, so I'll use our new prod instance once it's set up.